man/resolved.conf.xml

```diff
@@ -256,6 +256,22 @@
         in use.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>DNSStubListenerAddress=</varname></term>
+        <listitem><para>Takes a IPv4 address. Specifies the address which the DNS stub resolver will listen for requests
+        on port 53 depending on <varname>DNSStubListener=</varname> UDP, TCP or both. Default to unset.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>DNSStubListenerBindtoDevice=</varname></term>
```

Comment:
Maybe just `DNSStubListenerInterface=` ? This is shorter and avoids the tricky issue of whether "to" should be capitalized.

Reply:
yeah done.